src/scss/_typography.scss

```diff
@@ -0,0 +1,19 @@
+.h1 {
```

Comment:
repeating the sentiments from the HTML, if you can be more descriptive with this class name it can accomplish more. Try to keep the BEM formatting/style you are using elsewhere.